geode-junit/src/main/java/org/apache/geode/test/process/ProcessWrapper.java

```diff
@@ -86,6 +87,7 @@
   private Thread processThread;
   private ProcessStreamReader stdout;
   private ProcessStreamReader stderr;
+  private Consumer<String> consumer;
 
   private ProcessWrapper(final String[] jvmArguments, final Class<?> mainClass,
```

Comment:
Is the `Consumer` left out of the constructor intentionally so that you can control whether or not the `ProcessWrapper` logs or not?